catalog/hyperledger/multi-cluster.bom

```diff
@@ -263,6 +266,144 @@ brooklyn.catalog:
       stop.command: |
         sudo kill -9 $(cat $PID_FILE)
 
+  - id: hyperledger-fabric-composer-host-multi
+    description: |
+      A Docker host running with Fabric Composer installed.
+    itemType: entity
+    item:
+
+      name: "Fabric Composer Host"
+      type: hyperledger-docker-engine
+
+      brooklyn.children:
+      - type: hyperledger-fabric-composer-node-multi
+        id: my-hyperledger-fabric-composer-node
+
+  - id: hyperledger-fabric-composer-node-multi
+    description: "A Hyperledger Fabric Composer installation"
+    itemType: entity
+    item:
+
+      type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
+      name: "Fabric Composer Node"
+
+      brooklyn.config:
+        hyperledger.membersrvc: $brooklyn:component("my-hyperledger-membersrvc-node-multi").attributeWhenReady("host.address")
+        hyperledger.rootnode: $brooklyn:component("my-hyperledger-root-node-multi").attributeWhenReady("host.address")
+
+        templates.install:
+          "classpath://io.brooklyn.hyperledger:hyperledger/fabric-composer-templates/connection-template.json": "connection.json"
+
+        launch.latch: $brooklyn:component("my-hyperledger-root-node-multi").attributeWhenReady("service.isUp")
+
+      shell.env:
+        INSTALL_DIR: $brooklyn:attributeWhenReady("install.dir")
+
+      install.command: |
+        sudo yum -y update
+        sudo yum -y install git npm gcc-c++
+        sudo npm install -g composer-cli
+
+        git clone https://github.com/fabric-composer/sample-applications.git
+        cd sample-applications/packages/getting-started
+
+        sed -i 's|    "install": "scripts/download-hyperledger.sh && scripts/start-hyperledger.sh && npm run deployNetwork"||g' package.json
```

Comment:
I think it would be better to accept an `archiveFile` in a configKey and run `compose archive create --archiveFile` store the result.bna file in a sensor and then `composer network deploy attribeteWhenReady(file.bna)` wdyt?